File: code-examples/pymongo/collation/query-not-indexed-collation.py
from pymongo import MongoClient
from pymongo.collation import Collation

client = MongoClient('localhost', 27017)
db = client['database_name'] 
myColl = db['myColl']

myColl.find({"year": 1980}).sort("title", -1).collation(Collation(locale="en_US", strength=2))

File: code-examples/scala/collation/query-not-indexed-collation.scala
/*
   Retrieve documents that match the "year" value "1980"
   in descending order of the value of the "title" field,
   specifying a collation that does not use the collation
   index.
*/

// start not indexed collation
myColl.find(Filters.eq("year", 1980))
  .sort(Sorts.orderBy(Sorts.descending("title")))
  .collation(Collation.builder().locale("en_US").collationStrength(CollationStrength.SECONDARY).build())
// end not indexed collation
